manual/googlechrome-extensions/googlechrome-github-expandinizr/tools/install_x64.reg

```diff
@@ -0,0 +1,2 @@
+[HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432node\Google\Chrome\Extensions\cbehdjjcilgnejbpnjhobkiiggkedfib]
```

Comment:
@bcurran3 what OS are you running?  I am on Windows 8.1.  @AdmiringWorm what about you?